dali/test/python/test_operator_gaussian_blur.py

```diff
@@ -300,7 +300,16 @@ def test_per_sample_gaussian_blur():
                     yield check_per_sample_gaussian_blur, 10, sigma_dim, window_size_dim, shape, layout, axes, dev
 
 
-@raises(RuntimeError)
+@raises(RuntimeError,
+        regex="Only channel-first or channel-last layouts are supported, got: .*\.|"
+              "For sequences, layout should begin with 'F' or 'CF', got: .*\.|"
+              "Too many dimensions, found: \d+ data axes, maximum supported is: 3\.|"
+              "Found more the one occurrence of 'F' or 'C' axes in layout: .*\.|"
+              "`sigma` and `window_size` shouldn't be 0 at the same time for sample: \d+, axis: \d+\.|"
+              "`sigma` must have non-negative values, got .\d* for sample: \d*, axis: \d*\.|"
+              "`window_size` must have non-negative values, got .\d* for sample: \d*, axis : \d*\.|"
+              "Even or non-centered windows are not supported yet, got window with even length: [\s\S]* for sample \d*\.|"
+              "Kernel window should have odd length, got: \d*\.")
```

Comment:
In such case, instead of matching any of the alternatives to the error, other tended to use a list of globs and matching particular test case and error message like in dali/test/python/test_operator_squeeze.py for example.